project/plugins.sbt

```diff
@@ -1,5 +1 @@
-addSbtPlugin("com.dwijnand" % "sbt-dynver" % "2.0.0")
```

Comment:
What is `sbt-dynver` ?

Reply:
It used to set the release version for travis/bintray releases, as far as I understand. This will now be done by github releases and the `sbt-ci-release` plugin.